coffee-chats/src/main/java/com/google/step/coffee/JsonServlet.java

```diff
@@ -0,0 +1,93 @@
+package com.google.step.coffee;
+
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+import com.google.gson.annotations.Expose;
+
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+
+/**
+ *
+ * Provides an abstract class for JSON endpoints.
```

Comment:
Something is wrong with empty comment lines here.

https://google.github.io/styleguide/javaguide.html#s7-javadoc